packages/augur-ui/src/modules/auth/actions/update-assets.ts

```diff
@@ -72,8 +74,29 @@ export const updateAssets = (): ThunkAction<any, any, any, any> => async (
       })
     );
     let status = appStatus[WALLET_STATUS];
-    if (createBigNumber(dai).gt(FIVE) && (status !== WALLET_STATUS_VALUES.CREATED)) {
+    if (createBigNumber(daiBalance).gte(FIVE) && (status !== WALLET_STATUS_VALUES.CREATED)) {
       dispatch(updateAppStatus(WALLET_STATUS, WALLET_STATUS_VALUES.FUNDED_NEED_CREATE));
     }
+
+    const feeReserveAmount = createBigNumber(env.gsn.desiredSignerBalanceInETH);
+    const desiredEthBalance = feeReserveAmount.minus(signerEthBalance);
+    let activateFeeReserve = true;
+    if (desiredEthBalance.gt(ZERO)) {
+      // gasPrice
+      const gasPriceGwei = gasPriceInfo.userDefinedGasPrice || gasPriceInfo.average
+      const gasPriceWei = createBigNumber(gasPriceGwei).times(createBigNumber(GWEI_CONVERSION));
+
+      // convert worst cost tx in dai
+      const dai2EthConvert = createBigNumber(dai2Eth.value);
+      const desiredEthReserveInDai = feeReserveAmount.times(dai2EthConvert);
```

Comment:
feeReserveAmount here should be `desiredEthBalance `

Reply:
made change